myREADME.md

```diff
@@ -0,0 +1,14 @@
+Run
+```ruby
+$ ruby -r ./lib/order
+```
+
+The test menu used in development is:

```

Comment:
Mentioning the test menu is useful. I also would've like to have seen more information about your approach to this task, difficulties, etc